DataFormats/ForwardDetId/src/MTDChannelIdentifier.cc

```diff
@@ -0,0 +1,33 @@
+// Modify the pixel packing to make 100micron pixels possible. d.k. 2/02
+//
+#include "DataFormats/ForwardDetId/interface/MTDChannelIdentifier.h"
+
+
+/*
+// Extract from CMSIM manual (version Thu Jul 31 16:38:50 MET DST 1997)
+// --------------------------------------------------------------------
+// DIGI format for pixel
+//
+// For pixel digitization one word per fired pixel is used. 
+// The information includes pixel row and column number, time
+// and charge information with 7, 9, 4 and 12 bits for each as shown below. 
+//
+//  :DETD    :TRAK  :PXBD    4   #. no. of digitization elements
+//   #. name     no. bits
+//     :V          7             #. row number
+//     :W          9             #. column number
+//     :TIME       4             #. time (ns)
+//     :ADC       12             #. charge
+//
+// MODIFY 19.02.2002 for ORCA_6
```

Comment:
from Nov 2 https://github.com/cms-sw/cmssw/pull/25063/files#r230350346


Is this date reference appropriate?

Reply:
I removed the comment.